### ISSUE TITLE
Adding a new small section in the right bottom for the stocks rate/ news headlines 

### ISSUE THREAD

**Issue by shubhagarwal1** [SwanandD121/FeatherPerfect_fe#166]
We will be calling a free stock api to fetch the data and further dispaly it 
API: https://www.alphavantage.co/documentation/

**Comment by SwanandD121** [comment]
How is stock data relevant in a social media application made for wildlife enthusiasts?
Explain.

**Comment by shubhagarwal1** [comment]
> How is stock data relevant in a social media application made for wildlife enthusiasts? Explain.

@SwanandD121, yes you are correct, we can get latest wildlife news over there, which would be more insightful for our users.
